POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Views/onLaunchContentDialog.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<ContentDialog
+    x:Class="PowerToys_Settings_Sandbox.Views.onLaunchContentDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:PowerToys_Settings_Sandbox.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Welcome to PowerToys"
+    PrimaryButtonText="Go to tour"
+    SecondaryButtonText="Not now"
+    PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
+    SecondaryButtonClick="ContentDialog_SecondaryButtonClick"
+    DefaultButton="Primary">
+
+
+    <Grid>
+        <StackPanel>
+            <StackPanel>
+                <Image Source="ms-appx:///Assets/PowerToysAppList.targetsize-20.png" Width="70"/>
+                <TextBlock HorizontalAlignment="Center">Do more with PowerToys</TextBlock>
+            </StackPanel>
+            <StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
+                <StackPanel>
+                    <Image Source="ms-appx:///Assets/demo.gif" Width="70"/>
+                    <TextBlock HorizontalAlignment="Center">preview</TextBlock>
```

Comment:
Should also be capitalized like everything else?

Reply:
oh yeah. Ideally we want the "p" to be uppercase. Will correct that in then next iteration